client/openapi.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Gnerated things with open api tool seems no needed except *.ts.
So how we just use *.ts file only?